### ISSUE TITLE
zpool plugin doesn't work with ZFS >=0.8.0

### ISSUE THREAD

**Issue by Ninpo** [tobi-wan-kenobi/bumblebee-status#446]
#### Summary
Affected module: zpool

Description:
In 0.8.0 onwards, zpool list has an additional field, CKPOINT, breaking the parser in the zpool module.

#### What is the current behaviour and what is the expected behaviour?
Currently, for ZFS 0.8.0 onwards, no output is returned from the zpool module.
I expect to see ZFS pool information
#### How to reproduce
Run bumblebee-status -m zpool with ZFS >=0.8.0

I've created a pull request for a fix that should suffice and work across Python versions.
https://github.com/tobi-wan-kenobi/bumblebee-status/pull/445

Output from ZFS 0.6.5.6 on Ubuntu 16.04 with fix:

```bash
./bumblebee-status -m zpool -p zpool.sudo="true"
{"click_events": true, "version": 1}
[
[{"min_width": null, "separator": true, "background": null, "full_text": " pool sqldata ONL 681G/2.91T (78%) ", "instance": "6c3cb291-8a19-4aef-a254-ca218ff54a35", "color": "#aabbcc", "name": "zpool", "align": null, "separator_block_width": null}, {"min_width": null, "separator": true, "background": null, "full_text": " pool read  114.54KiB ", "instance": "f42e5c04-054c-4980-9585-a9501e33d1c3", "color": "#aabbcc", "name": "zpool", "align": null, "separator_block_width": null}, {"min_width": null, "separator": true, "background": null, "full_text": " pool write  79.23KiB  ", "instance": "aa38e517-0af7-4c68-9f08-2240969a50ed", "color": "#aabbcc", "name": "zpool", "align": null, "separator_block_width": null}]
```

Output from ZFS master branch on Gentoo with fix:
```bash
$ python bumblebee-status -m zpool -p zpool.sudo="true"
{"version": 1, "click_events": true}
[
[{"full_text": " pool boot ONL 47.0M/480M (91%) ", "color": "#aabbcc", "background": null, "separator_block_width": null, "separator": true, "min_width": null, "align": null, "instance": "aebebb48-9bfa-42eb-9e65-2e64bab8bc48", "name": "zpool"}, {"full_text": " pool read  0.02B     ", "color": "#aabbcc", "background": null, "separator_block_width": null, "separator": true, "min_width": null, "align": null, "instance": "7e8112ff-8b04-4ec8-ad69-e8c21ba6aa03", "name": "zpool"}, {"full_text": " pool write  0.00B     ", "color": "#aabbcc", "background": null, "separator_block_width": null, "separator": true, "min_width": null, "align": null, "instance": "2ba51163-8cb5-4320-91be-9a013e263254", "name": "zpool"}, {"full_text": " pool rpool ONL 36.3G/464G (93%) ", "color": "#aabbcc", "background": null, "separator_block_width": null, "separator": true, "min_width": null, "align": null, "instance": "a6ee4617-ca7b-4c64-be12-0e81dbe6f4c6", "name": "zpool"}, {"full_text": " pool read  54.38B    ", "color": "#aabbcc", "background": null, "separator_block_width": null, "separator": true, "min_width": null, "align": null, "instance": "fd38dbf3-6942-4b09-83cd-2075c5477781", "name": "zpool"}, {"full_text": " pool write  63.47B    ", "color": "#aabbcc", "background": null, "separator_block_width": null, "separator": true, "min_width": null, "align": null, "instance": "00ee5ddd-7bd3-4576-ae79-cb7d3289b3f5", "name": "zpool"}]
```
Output from ZFS master branch on Gentoo without fix:
```bash
$ python bumblebee-status -m zpool -p zpool.sudo="true"
{"version": 1, "click_events": true}
[
[],
```

Edit: I see some work needs doing to get tests to pass, I'll take a look at that as soon as I can.

**Comment by Ninpo** [comment]
PR Updated

**Comment by tobi-wan-kenobi** [comment]
Should be fixed by merging PR #445
